aws/resource_aws_batch_job_definition_test.go

```diff
@@ -103,6 +104,10 @@ func TestAccAWSBatchJobDefinition_ContainerProperties_Advanced(t *testing.T) {
 		},
 		RetryStrategy: &batch.RetryStrategy{
 			Attempts: aws.Int64(int64(1)),
+			EvaluateOnExit: []*batch.EvaluateOnExit{
+				{Action: aws.String(strings.ToLower(batch.RetryActionRetry)), OnStatusReason: aws.String("Host EC2*")},
```

Comment:
I applied `strings.ToLower` on the `Action` because it appears to come back lowercase from the AWS API.